packages/wrangler/src/api/startDevWorker/types.ts

```diff
@@ -94,13 +117,27 @@ export interface StartDevWorkerOptions {
 		outboundService?: ServiceFetch;
 		/** An undici MockAgent to declaratively mock fetch calls to particular resources. */
 		mockFetch?: undici.MockAgent;
+
+		/** Gets a fetcher to a specific worker, used for multi-worker development */
+		getRegisteredWorker?(name: string): WorkerDefinition | undefined;
 	};
 }
 
-export type Hook<T, Args extends unknown[] = unknown[]> =
+export type Hook<T extends string | number | object> =
 	| T
 	| Promise<T>
-	| ((...args: Args) => T | Promise<T>);
+	| (() => T | Promise<T>);
+
+export type Module<ModuleType extends ModuleRule["type"] = ModuleRule["type"]> =
+	File<string | Uint8Array> & {
+		/** Name of the module, used for module resolution, path may be undefined if this is a virtual module */
+		name: string;
+		/** How this module should be interpreted */
+		type: ModuleType;
+	};
+
+// TODO: revisit this type
```

Comment:
when?

Reply:
This is meant for later—I'll clarify

Reply:
I've removed this comment—we might want to update this type in the future, but we can address that if needed. It's definitely not a `TODO(now)` urgency refactoring

Reply:
https://github.com/cloudflare/workers-sdk/pull/4450/commits/44c727e39c8560cc960f720529505a393ff2de53